src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -155,36 +161,55 @@ foam.CLASS({
     var menuId = this.currentMenu ? this.currentMenu.id : this.config.of.id;
     this.addClass(this.myClass())
 
-      .add(this.slot(function(data, config, config$of, config$browseBorder, config$browseViews, config$browseTitle, config$browseSubtitle, config$primaryAction, config$createTitle, config$createControllerView) {
+      .add(this.slot(function(data, config, config$of, config$browseBorder, config$browseViews, config$browseTitle, config$browseSubtitle, config$primaryAction, config$createTitle, config$createControllerView, config$browseContext) {
         return self.E()
           .start(self.Rows)
             .addClass(self.myClass('container'))
               .start()
                 .addClass(self.myClass('header-container'))
                 .start(self.Cols)
                   .start()
-                    .addClasses(['h100',self.myClass('browse-title')])
+                    .addClasses(['h100', self.myClass('browse-title')])
                     .translate(menuId + ".browseTitle", config$browseTitle)
                   .end()
-                  .callIf( ! config.detailView, function() {
-                    this.startContext({ data: self })
-                      .tag(self.CREATE, {
-                           label: this.translationService.getTranslation(foam.locale, menuId + '.createTitle', config$createTitle),
-                           buttonStyle: foam.u2.ButtonStyle.PRIMARY
-                      })
-                    .endContext()
-                  })
-                  .callIf( config.createControllerView, function() {
-                    this.startContext({ data: self })
-                      .tag(self.CREATE, {
-                           label: this.translationService.getTranslation(foam.locale, menuId + '.handler.createControllerView.view.title', config$createControllerView.view.title),
-                           buttonStyle: foam.u2.ButtonStyle.PRIMARY
-                      })
-                    .endContext()
-                  })
-                  .callIf( config$primaryAction, function() {
-                    this.startContext({ data: self }).tag(config$primaryAction, { size: 'LARGE', buttonStyle: 'PRIMARY' }).endContext();
-                  })
+                  .start(self.Cols)
+                    .callIf( config.browseActions != [] && config.browseContext, function() {
```

Comment:
I don't think this comparison works. Try config.browseActions.length && config.browseContext.

Reply:
Yup, didn't realise that. Fixed.